lib/autopilot/resources/contact.rb

```diff
@@ -7,5 +7,6 @@ class Contact < Resource
     include Operations::All
     include Operations::Retrieve
     include Operations::Identify
+    include Operations::TriggerJourney
```

Comment:
@lcpriest what you mean for non restful here? There is no journey resource and the only thing you can trigger is a jorney. But I feel something is wrong with this modules anyway.